es6-promises/main.ts

```diff
@@ -0,0 +1,9 @@
+import { takeAChance } from './take-a-chance.js';
+
+takeAChance('Tiffany')
+  .then((resolve: string) => {
+    console.log(resolve);
+  })
+  .catch((error) => {
+    console.log(error.message);
+  });
```

Comment:
Since the function bodies have just one statement, you don't need `{}`.

Also, TypeScript can figure out the type of the function parameter, so you don't need to specify it.